spk/flexget/Makefile

```diff
@@ -40,11 +40,9 @@ include ../../mk/spksrc.python.mk
 # [greenlet]
 ifeq ($(call version_ge, $(TC_GCC), 5.0),1)
 WHEELS += src/requirements-crossenv-greenlet-v3.txt
-else ifeq ($(call version_ge, $(TC_GCC), 4.9)$(call version_lt, $(TC_GCC), 5.0),11)
+else
 WHEELS += src/requirements-crossenv-greenlet-v3-gcc4.txt
 WHEELS_CPPFLAGS += [greenlet] -std=c++11 -fpermissive
-else
-WHEELS += src/requirements-crossenv-greenlet-v1.txt
 endif
 
 # [Pendulum]
```

Comment:
I suggest moving pendulum to python subdirectory.

Reply:
Good idea, will open a new PR for that as this will require updating all of your `python31x-wheels` `Makefile` files as well.